react/components/ProductTitleAndPixel.tsx

```diff
@@ -123,6 +124,7 @@ function getSkuProperties(item: SKU): SKUEvent {
         Price: seller.commertialOffer.Price,
         ListPrice: seller.commertialOffer.ListPrice,
         AvailableQuantity: seller.commertialOffer.AvailableQuantity,
+        PriceWithoutDiscount: seller.commertialOffer?.PriceWithoutDiscount || 0,
```

Comment:
If `commertialOffer` can be null/undefined, wouldn't you need to add `optionalChaining (?)` to the other attributes?